pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+comp.lang.PyLadies newsgroup (a.k.a. PyLadies-list@PyLadies.org mailing
+list) or the PyLadies-ideas mailing list is the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a POP is meant
+to save the potential author time. Many ideas have been brought
+forward for changing PyLadies that have been rejected for various
+reasons. Asking the PyLadies community first if an idea is original
+helps prevent too much time being spent on something that is
+guaranteed to be rejected based on prior discussions (searching
+the internet does not always do the trick). It also helps to make sure
+the idea is applicable to the entire community and not just the author.
+Just because an idea sounds good to the author does not
+mean it will work for most people in most areas where PyLadies is used.
+
+Once the champion has asked the PyLadies community as to whether an
+idea has any chance of acceptance, a draft POP should be presented to
+PyLadies-ideas.  This gives the author a chance to flesh out the draft
+POP to make properly formatted, of high quality, and to address
+initial concerns about the proposal.
+
+Following a discussion on PyLadies-ideas, the proposal should be sent as a
+draft POP to the POP editors <POPs@PyLadies.org>.  The draft must be written
+in POP style as described below, else it will be sent back without further
+regard until proper formatting rules are followed (although minor errors
+will be corrected by the editors).
+
+If the POP editors approve, they will assign the POP a number, label it
+as Standards Track, Informational, or Process, give it status "Draft",
+and create and check-in the initial draft of the POP.  The POP editors
+will not unreasonably deny a POP.  Reasons for denying POP status
+include duplication of effort, being technically unsound, not
+providing proper motivation or addressing backwards compatibility, or
+not in keeping with the PyLadies philosophy.  The BDFL can be consulted
+during the approval phase, and is the final arbiter of the draft's
+POP-ability.
+
+Developers with hg push privileges for the `POP repository`_ may claim
+POP numbers directly by creating and committing a new POP. When doing so,
+the developer must handle the tasks that would normally be taken care of by
+the POP editors (see `POP Editor Responsibilities & Workflow`_). This
+includes ensuring the initial version meets the expected standards for
+submitting a POP. Alternately, even developers may choose to submit POPs
+through the POP editors. When doing so, let the POP editors know you have
+hg push privileges and they can guide you through the process of updating
+the POP repository directly.
+
+As updates are necessary, the POP author can check in new versions if they
+(or a collaborating developer) have hg push privileges, or else they can
+email new POP versions to the POP editors for publication.
+
+After a POP number has been assigned, a draft POP may be discussed further on
```

Comment:
I think, with our workflow of:

1. author proposing idea
2. author submitting PR
3. POP editors and others comment on PR (and any necessary edits made by the author)
4. POP editor locally pulls it & edits status/label/etc
5. POP editor merges to master

That we don't need to discuss it further elsewhere.